.github/workflows/flink-ci.yml

```diff
@@ -19,27 +19,13 @@
 
 name: "Flink CI"
 on:
-  push:
```

Comment:
This would stop running CI on the master branch. Just needs another attribute:
```
    branches: [ master ]
```

Reply:
I was going to make a separate PR that would run the full test suite on master (so merges to master don't get preempted), but for now that's a better idea.

Reply:
I'm going to remove the `paths-ignore`, as when I spoke to @rdblue about this a while ago, the ideal situation was to run all tests after merging to master.

That way, we don't have to keep both lists synchronized. Then eventually we can have a separate PR workflow just for merging to master and for tagging (new releases).

Reply:
For master/release-branch builds that makes sense to me.

Reply:
Ok. I added the following which should cover any tags.
```yaml
on:
  push:
    branches:
    - 'master'
   tags:
   - '**'
```